src/components/audio-player-view/audio-player-view.scss

```diff
@@ -1,10 +1,17 @@
-.audio-player-view {
+:root {
+  --ma-player-border-radius: 8px;
+}
+
+$ma-player-border-radius: var(--ma-player-border-radius);
+
+.mini-audio-player-view {
   position: relative;
   background-color: var(--playkit-paper-color);
   margin: 0;
   min-width: 228px;
-  border-radius: 8px;
+  border-radius: $ma-player-border-radius;
   justify-content: space-evenly;
+  //aspect-ratio: 10 / 3;
```

Comment:
commented leftovers

Reply:
Intently there for me - it is refs for me which are critically relevant for the next task on mini-audio responsiveness